configuration/editors.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: documentation
+title: Editors
+---
+
+{% include base.html %}
+
+# Editors - Different ways to power up your textual configuration
+
+OpenHAB has a great development environment with the "openHAB Designer".
+But in some cases you want to use another editor to make changes to the configuration files of openHAB.
+To make this more effectiv there are some files to enable syntax highlighting for openHAB-files in these editors.
+
+{::options toc_levels="2"/}
+
+* TOC
+{:toc}
+
+## mcedit
+
+mcedit is an editor which comes with mc (Midnight Commander).
+
+
+### Installing the syntax-files
+
+- download copy the syntax-files to */usr/share/mc/syntax/*
+- https://code.google.com/p/openhab-samples/source/browse/syntaxhl/mc/?repo=wiki
```

Comment:
Hey, found it! :smiley: 
https://groups.google.com/forum/#!searchin/openhab/syntax$20highlighting%7Csort:relevance/openhab/zJrfTDjahbc/r8GcpKDgr1cJ